packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,313 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+import { Step, StepProps, StepStatus } from './Step';
+import { transparentize } from 'polished';
+
+/**
+ * @children required
+ */
+export interface StepperProps extends React.HTMLAttributes<HTMLDivElement> {
+  testId?: string;
+  /*
+   * Custom number for specific breakpoint responsive styling
+   */
+  breakpoint?: number;
+  /*
+   * Determines the Stepper view based on a minimum breakpoint number.
+   */
+  breakpointStyle?: BreakPointStyle;
```

Comment:
I don't understand this prop and enum. Maybe we can chat about this.

Reply:
Now named `breakpointLayout` per our discussion.